platform/src/default.rs

```diff
@@ -23,6 +34,49 @@ pub const TEST_CERT_PEM: &[u8] = include_bytes!("test_data/cert_256.pem");
 #[cfg(feature = "dpe_profile_p384_sha384")]
 pub const TEST_CERT_PEM: &[u8] = include_bytes!("test_data/cert_384.pem");
 
+impl DefaultPlatform {
+    cfg_if! {
+        if #[cfg(feature = "openssl")] {
+        fn parse_issuer_name() -> Vec<u8> {
+            X509::from_pem(TEST_CERT_PEM)
+                .unwrap()
+                .subject_name()
+                .to_der()
+                .unwrap()
+        }} else if  #[cfg(feature = "rustcrypto")] {
+               fn parse_issuer_name() -> Vec<u8> {
+            Certificate::from_pem(TEST_CERT_PEM)
+                .unwrap()
+                .tbs_certificate
+                .subject
+                .to_der()
+                .unwrap()
+        }
+    }}
+
+    cfg_if! {
+        if #[cfg(feature = "openssl")] {
+       fn parse_issuer_sn() -> Vec<u8> {
+            X509::from_pem(TEST_CERT_PEM)
+                .unwrap()
+                .serial_number()
+                .to_bn()
+                .unwrap()
+                .to_vec()
+        }
+        } else if  #[cfg(feature = "rustcrypto")] {
+          fn parse_issuer_sn() -> Vec<u8> {
+            Certificate::from_pem(TEST_CERT_PEM)
+                .unwrap()
+                .tbs_certificate
+                .serial_number
+                .as_bytes()
+                .to_vec()
+        }
+        }
+    }
+}
+
```

Comment:
optional: this might just be personal preference, but I think if they are separated into distinct modules it might be a little more readable.
```suggestion
pub use default::DefaultPlatform
#[cfg(feature = "openssl")]
mod default {
    use super::*;
    use openssl::x509::X509;

    pub struct DefaultPlatform;
    impl DefaultPlatform {
        pub fn parse_issuer_name() -> Vec<u8> {
            X509::from_pem(TEST_CERT_PEM)
                .unwrap()
                .subject_name()
                .to_der()
                .unwrap()
        }

        pub fn parse_issuer_sn() -> Vec<u8> {
            X509::from_pem(TEST_CERT_PEM)
                .unwrap()
                .serial_number()
                .to_bn()
                .unwrap()
                .to_vec()
        }
    }
}

#[cfg(feature = "rustcrypto")]
mod default {
    use super::*;
    use x509_cert::{
        certificate::Certificate,
        der::{DecodePem, Encode},
    };

    impl DefaultPlatform {
        pub fn parse_issuer_name() -> Vec<u8> {
            Certificate::from_pem(TEST_CERT_PEM)
                .unwrap()
                .tbs_certificate
                .subject
                .to_der()
                .unwrap()
        }

        pub fn parse_issuer_sn() -> Vec<u8> {
            Certificate::from_pem(TEST_CERT_PEM)
                .unwrap()
                .tbs_certificate
                .serial_number
                .as_bytes()
                .to_vec()
        }
    }
}
```

Reply:
Sent #288 (left cfg_if in to allow clippy/fmt to run on both impls by default)